programs/bpf_loader/src/lib.rs

```diff
@@ -201,8 +201,10 @@ macro_rules! deploy_program {
             Arc::new(program_runtime_environment),
         )?;
         if let Some(old_entry) = find_program_in_cache($invoke_context, &$program_id) {
-            let usage_counter = old_entry.usage_counter.load(Ordering::Relaxed);
-            executor.usage_counter.store(usage_counter, Ordering::Relaxed);
+            let usage_counter = old_entry.tx_usage_counter.load(Ordering::Relaxed);
```

Comment:
I think it would make more sense to just inline these as temporary values instead of having two variables named `usage_counter`.